CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -88,7 +88,7 @@ void CaloTPGTranscoderULUT::loadHCALCompress(HcalLutMetadata const& lutMetadata,
         }
     } else {
       for (unsigned int i = threshold; i < lutsize; ++i)
-        outputLUT_[index][i] = version == 0 ? linearRctLUT[i] : linearNctLUT[i];
+        outputLUT_[index][i] = version == 1 ? linearNctLUT[i] : linearRctLUT[i];
```

Comment:
I don't understand these changes. Why would the output LUT need to change between version 1 and version 2?

Reply:
I wanted v2 TPs to be associated with Rct like v0 TPs are since the choice here does not depend whether v0 or v2 for HBHE TPs. Likewise, Nct is associated with v1, so I "flipped" the logic to explicitly associate v1 with Nct and anything else (v0 or v2) with Rct. However, maybe I am misunderstanding other implications of this.